rfc/0009-qsl-structure.md

```diff
@@ -60,13 +60,11 @@ Keep in mind that the modules listed here are the names of the current Fabric AP
         - (future) Proper Handshake API
     - Crash Reports
     - Resource Loader
-- `common` - "Dumping ground" of shared code needed across modules. This is a little more complex, and uses github's code owners to define ownership of individual modules instead of the library as a whole. Modules should only be here if 100% needed, and if enough similar code goes here it can be split to its own library.
-	- None currently
 - `transfer`
     - API Lookup API
     - fluid api
     - item transfer api
-- `config_and_admin` // better name needed
+- `command` // This really is anything server-side only related to administration, but "command" is a nice word to describe the primary use-case.
     - Command API
     - Game Rule API
```

Comment:
I'm not sure this is appropriate still.

Given there is a game rule screen and it really has a world scope.